cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -64,7 +64,9 @@
   public static final String JMS_MESSAGES_TO_RECEIVE = "jms.messages.receive";
   public static final String JMS_NAMING_FACTORY_INITIAL = "jms.factory.initial";
   public static final String JMS_PROVIDER_URL = "jms.provider.url";
-  public static final String JMS_CONNECTION_FACTORY_NAME = "ConnectionFactory";
+  public static final String JMS_CONNECTION_FACTORY_NAME = "jms.jndi.connectionfactory.name";
+
+  public static final String CONNECTION_FACTORY = "ConnectionFactory";
```

Comment:
This is a string constant to just save some time to retype so usually the variable name is exactly the same as the String it represents